src/trex_timesync.cpp

```diff
@@ -141,7 +135,7 @@ int64_t CTimesyncEngine::evalDelta(int port, uint16_t sequence_id) {
 }
 
 void CTimesyncEngine::setDelta(int port, int64_t delta) {
-    printf("PTP delta for port %d: %ld ns\n", port, delta);  // TODO this is useful but not mandatory, shall we keep it?
+    printf("PTP delta for port %d: %ld ns\n", port, delta);
```

Comment:
Maybe add some #ifdefine debug?

Reply:
Well, … no.  I thought about it.  And as PTP is (at this moment at least) so crucial to out modifications, I would leave this `printf` as it is